### ISSUE TITLE
Copy/Paste does not work in the terminal

### ISSUE THREAD

**Issue by JonasHelming** [eclipse-theia/theia-blueprint#121]
@lucas-koehler reported that Copy/Paste does not work for him in the Terminal. I just retested this, it works for me, so this is potentially related to some environment variables.

**Comment by vince-fugnitto** [comment]
@JonasHelming it would be good if he can report with as much information as possible, including the operating system, and shell used.

**Comment by lucas-koehler** [comment]
I use Ubuntu 20.04 with ZSH including [oh my ZSH](https://github.com/ohmyzsh/ohmyzsh)

**Comment by lucas-koehler** [comment]
@vince-fugnitto  @JonasHelming Because it worked for Jonas I bruteforce tried it out again and I noticed that double right-clicking opens the context menu and then it showed me the shortcuts. I just didn't expect a double right-click to be necessary and the short cuts to be different.
Usually, its CTRL+SHIFT+C and CTRL+SHIFT+V for me but in the blueprint terminal it's just CTRL+C and CTRL+V.

**Comment by JonasHelming** [comment]
@lucas-koehler Do you still consider this as a bug or shall we close this?

**Comment by lucas-koehler** [comment]
@JonasHelming I would consider it a minor bug but I do not think it's worth investigating it because the different shortcuts could also be seen as a feature (consistent shortcuts in theia terminal independently of the native terminal).
The right click menu I would only investigate if there are more reports of this because it might be some strange interaction between ZSH and the theia terminal that might be hard to locate and/or fix.
In conclusion, I would close it.

**Comment by JonasHelming** [comment]
@lucas-koehler thanks!

**Comment by vince-fugnitto** [comment]
I'll open an issue upstream about the double-right-click to open the context-menu which I can reproduce with vanilla theia.

**Comment by JonasHelming** [comment]
@vince-fugnitto : Thank you!
